docs/standard/generics/math.md

```diff
@@ -0,0 +1,199 @@
+| <xref:System.Numerics.IUnaryPlusOperators%602>     | `+x`                                     |
+
+> [!NOTE]
+> Some of the interfaces define a checked operator in addition to a regular unchecked operator. Checked operators are called in checked contexts and allow a user-defined type to define overflow behavior. If you implement a checked operator, for example, <xref:System.Numerics.ISubtractionOperators%603.op_CheckedSubtraction(%600,%601)>, you must also implement the unchecked operator, for example, <xref:System.Numerics.ISubtractionOperators%603.op_Subtraction(%600,%601)>.
+
+### Function interfaces
+
+The function interfaces define common mathematical APIs that apply more broadly than to a specific [numeric interface](#numeric-interfaces). These interfaces are all implemented by <xref:System.Numerics.IFloatingPointIeee754%601>, and may get implemented by other relevant types in the future.
+
+| Interface name                                     | Description                                                                                                              |
+|----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| <xref:System.Numerics.IExponentialFunctions%601>   | Exposes exponential functions supporting `e^x`, `e^x - 1`, `2^x`, `2^x - 1`, `10^x`, and `10^x - 1`.                     |
+| <xref:System.Numerics.IHyperbolicFunctions%601>    | Exposes hyperbolic functions supporting `acosh(x)`, `asinh(x)`, `atanh(x)`, `cosh(x)`, `sinh(x)`, and `tanh(x)`.         |
+| <xref:System.Numerics.ILogarithmicFunctions%601>   | Exposes logarithmic functions supporting `ln(x)`, `ln(x + 1)`, `log2(x)`, `log2(x + 1)`, `log10(x)`, and `log10(x + 1)`. |
+| <xref:System.Numerics.IPowerFunctions%601>         | Exposes power functions supporting `x^y`.                                                                                |
+| <xref:System.Numerics.IRootFunctions%601>          | Exposes root functions supporting `cbrt(x)` and `sqrt(x)`.                                                               |
+| <xref:System.Numerics.ITrigonometricFunctions%601> | Exposes trigonometric functions supporting `acos(x)`, `asin(x)`, `atan(x)`, `cos(x)`, `sin(x)`, and `tan(x)`.            |
+
+### Parsing interfaces
+
+Parsing is a common programming concept. It's typically used to support converting user input to a specified type, or deserializing data from a file. These interfaces are in the <xref:System> namespace.
+
+| Interface name                     | Description                                                                                      |
+|------------------------------------|--------------------------------------------------------------------------------------------------|
+| <xref:System.IParsable%601>        | Exposes support for `T.Parse(string, IFormatProvider)`.                                          |
```

Comment:
I left out the two formattable interfaces since I didn't see them anywhere in dotnet/runtime.

Reply:
`IFormattable` and `ISpanFormattable` are also implemented by all number (and many of the other) types. They represent the inverse operation from `IParsable` and allow generating a formatted string using the `ToString()` APIs.

Reply:
@tannergooding But they aren't new, correct? And they aren't generic?